src/components/ComponentPages/Login/loginModal.tsx

```diff
@@ -11,6 +11,7 @@ const LoginModal = () => {
   return (
     <Fragment>
       <Modal
+        style={{ top: "20px" }}
         visible={visible}
         // title="Title"
```

Comment:
Remove the commented code if not required

Reply:
removed